src/status_im/contexts/communities/actions/airdrop_addresses/view.cljs

```diff
@@ -19,18 +20,22 @@
           (fn []
             (if can-edit-addresses?
               (rf/dispatch
-               [:password-authentication/show
-                {:content (fn [] [password-authentication/view])}
-                {:label    (i18n/label :t/enter-password)
-                 :on-press (fn [password]
-                             (rf/dispatch
-                              [:communities/edit-shared-addresses
-                               {:community-id    community-id
-                                :password        password
-                                :airdrop-address address
-                                :on-success      (fn []
-                                                   (rf/dispatch [:dismiss-modal :address-for-airdrop])
-                                                   (rf/dispatch [:hide-bottom-sheet]))}]))}])
+               [:standard-auth/authorize
+                {:auth-button-label (i18n/label :t/confirm-changes)
+                 :on-auth-success   (fn [password]
+                                      (rf/dispatch
+                                       [:communities/edit-shared-addresses
+                                        {:community-id    community-id
+                                         :password        (security/safe-unmask-data password)
+                                         :airdrop-address address
+                                         :on-success      (fn []
+                                                            (rf/dispatch
+                                                             [:dismiss-modal
+                                                              :address-for-airdrop])
+                                                            (rf/dispatch
+                                                             [:hide-bottom-sheet]))}]))
+                 :on-auth-fail      (fn [err]
+                                      (log/info "Biometric authentication failed" err))}])
```

Comment:
The error is already logged in the `:standard-auth/on-biometric-fail` event. Same for all the other usages of `:on-auth-fail` in this PR.

Reply:
So that means we don't need an on-auth-fail on most usages of `standard-auth/authorize`, NIce!

Reply:
Yeah, it's in case you have some different behavior if auth failed. The logging is handled by the event.